utils/spock_create_subscriber/spock_create_subscriber.c

```
@@ -0,0 +1,1751 @@
+{
+	FILE	   *pidf;
+	long		pid;
+
+	pidf = fopen(pid_file, "r");
+	if (pidf == NULL)
+	{
+		return 0;
+	}
+	if (fscanf(pidf, "%ld", &pid) != 1)
+	{
+		return 0;
```

Cppcheck (2.21.0)

[error] Resource leak

(resourceLeak)